tests/lib/hqstyle-core.css

```diff
@@ -293,19 +293,13 @@ th {
 
 @font-face {
     font-family: 'Glyphicons Halflings';
-    src:
-url('https://dnwn0mt1jqwp0.cloudfront.net/static/bootstrap/fonts/glyphicons-halflings-regular.eot?f4769f9bdb74');
-    src:
-url('https://dnwn0mt1jqwp0.cloudfront.net/static/bootstrap/fonts/glyphicons-halflings-regular.eot?&f4769f9bdb74#iefix')
-format('embedded-opentype'),
-url('https://dnwn0mt1jqwp0.cloudfront.net/static/bootstrap/fonts/glyphicons-halflings-regular.woff2?448c34a56d69')
-format('woff2'),
-url('https://dnwn0mt1jqwp0.cloudfront.net/static/bootstrap/fonts/glyphicons-halflings-regular.woff?fa2772327f55')
-format('woff'),
-url('https://dnwn0mt1jqwp0.cloudfront.net/static/bootstrap/fonts/glyphicons-halflings-regular.ttf?e18bbf611f2a')
-format('truetype'),
-url('https://dnwn0mt1jqwp0.cloudfront.net/static/bootstrap/fonts/glyphicons-halflings-regular.svg?89889688147b#glyphicons_halflingsregular')
-format('svg')
+    src: url('/static/style/lib/bootstrap-3.2.0/fonts/glyphicons-halflings-regular.eot?f4769f9bdb74');
```

Comment:
When I load vellum stand-alone I now get two 404 errors that I did not get before:
```
GET http://localhost:8081/static/style/font/commcarehqfont-regular.woff?v=1.1&d68065ecb660 
GET http://localhost:8081/static/style/font/commcarehqfont-regular.ttf?v=1.1&b7dbbd6d243e 
```

Is there a way to make it so these errors do not occur? I don't like to get into a state where I'm ignoring errors in my dev environment.